proposals/open-telemetry/microprofile-opentelemetry-tracing.asciidoc

```diff
@@ -29,14 +29,26 @@ portability and the burden on the user to maintain the instrumentation.
 The https://opentelemetry.io[OpenTelemetry] project solves these problems by providing a single, vendor-agnostic
 solution.
 
-== Architecture
+== Rationale
 
-https://opentelemetry.io[OpenTelemetry] is a set of APIs, SDKs, tooling and integrations that are designed for the
-creation and management of telemetry data such as traces, metrics, and logs.
+https://opentelemetry.io[OpenTelemetry] is a set of APIs, SDKs, tooling, and integrations designed to create and manage
+telemetry data such as traces, metrics, and logs.
 
 This specification defines the behaviors that allow MicroProfile applications to easily participate in an environment
-where distributed tracing is enabled via https://opentelemetry.io[OpenTelemetry] (a merger between
-https://opentracing.io[OpenTracing] and https://opencensus.io[OpenCensus]).
+where distributed tracing is enabled via https://opentelemetry.io[OpenTelemetry].
+
+https://opentelemetry.io[OpenTelemetry] is a https://cncf.io/[CNCF] incubating project.
+Formed through a merger of the https://opentracing.io[OpenTracing] and https://opencensus.io[OpenCensus] projects.
+https://opentelemetry.io[OpenTelemetry] is adopted and supported by https://opentelemetry.io/vendors[industry leaders]
+in the observability space.
+
+Most programming languages support https://opentelemetry.io[OpenTelemetry], including Java, C++, .NET, Go,
+JavaScript, PHP, Python, Ruby, Rust and Swift.
+
+The MicroProfile community will participate in the https://opentelemetry.io[OpenTelemetry] specification and
```

Comment:
I feel this future tense is strange in the spec doc. I prefer to say:
It is best for the MicroProfile community to enage with the https://opentelemetry.io[OpenTelemetry] on its specification and Java implementation, rather than fork the OpenTelemetry APIs.

Reply:
Ok